Services/Administration/OJS.Services.Administration.Business/Implementations/ProblemsBusinessService.cs

```diff
@@ -176,6 +176,21 @@ public async Task<bool> UserHasProblemPermissions(int problemId, string? userId,
             return await this.contestsBusiness.UserHasContestPermissions(problem.ContestId, userId, isUserAdmin);
         }
 
+        public async Task ReevaluateProblemsByOrderBy(int contestId)
+        {
+            var orderByIndex = 0;
+
+            await this.contestsData.GetProblemsById(contestId)
+                                    .OrderBy(p => p.OrderBy)
```

Comment:
Each Problem is in ProblemGroup. Don't we need to reevaluate the order of the ProblemGroups as well?

Reply:
I have added an additional orderBy reevaluation of problems group in ProblemGroupController